ls_0/index.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/ls_0/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+
+        <header>
+            <nav>
+                <div class="nav-main">
+                    <div class="nav-main-left"><img src="/ls_0/src/images/Union.svg" alt="logo"></div>
+                    <div class="nav-main-right">Explore</div>
+                    <div class="nav-menu">
+                        <a href="#" class="nav-list">Сategories</a>
+                        <a href="#" class="nav-list">Сourse</a>
+                        <a href="#" class="nav-list">Explore</a>
+                        <a href="#" class="nav-list">Degrees</a>
+                    </div>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section class="title-sec">
+                <h1>Education is the <br>
+                    most powerfull <br>
+                    weapon</h1>
+                <p class="title-p">A bank is a financial institution accepts deposits </p>
```

Comment:
Text alining and wrapping should setted by container width and styles in future, not by separating by tags. Imagine you have dynamic taxt from the server. You can`t predict how many line it will have. But you can cover all valid cases by css styles. So for now use just one p tag for the whole text. Also change it in all similar places, please.